lib/fathead/diveintopython/output.txt

```diff
@@ -0,0 +1,24 @@
+Coercing Integers To Floats And Vice-Versa	A							http://www.diveintopython3.net/native-datatypes.html			As you just saw, some operators (like addition) will coerce integers to floating point numbers as needed. You can also coerce them by yourself.<br><pre><code>\n>>> float(2) \n2.0\n>>> int(2.0) \n2\n>>> int(2.5) \n2\n>>> int(-2.5) \n-2\n>>> 1.12345678901234567890 \n1.1234567890123457\n>>> type(1000000000000000) \n<class 'int'></code></pre>	http://www.diveintopython3.net/native-datatypes.html#number-coercion
```

Comment:
Triggering for these is on the exact key. You could increase the triggering by creating redirect entries with other verb tenses, `coercing to coerce`. Same idea for a lot of the other entries.
```
coerce integers to floats and view-versa    R  coercing integers to floats and vice-versa
````